.flake8

```diff
@@ -12,5 +12,5 @@ max-complexity = 12
 per-file-ignores =
 	docs/_api/conf.py: E265
 	integration-tests/steps/*: E501,F811,F403,F405
-extend-ignore = E203
+extend-ignore = E201,E202,E203,E231
```

Comment:
[Flake8](https://flake8.pycqa.org/en/latest/user/error-codes.html) told me to check [pycodestyle](https://pycodestyle.pycqa.org/en/latest/intro.html#error-codes) for the meaning of the codes.

* E201: whitespace after `(`
* E202: whitespace before `)`
* E231: missing whitespace after `,`, `;`, or `:`

Why do you add these just now?